README.md

```diff
@@ -1,7 +1,128 @@
 # KerasCV
-KerasCV is a repository of modular building blocks (layers, metrics, losses, data-augmentation) that applied computer vision engineers can leverage to quickly assemble production-grade, state-of-the-art training and inference pipelines for common use cases such as image classification, object detection, image segmentation, image data augmentation, etc. 
+KerasCV is a repository of modular building blocks (layers, metrics, losses, data-augmentation) that
+applied computer vision engineers can leverage to quickly assemble production-grade, state-of-the-art
+training and inference pipelines for common use cases such as image classification, object detection, 
+image segmentation, image data augmentation, etc. 
 
-KerasCV can be understood as a horizontal extension of the Keras API: they're new first-party Keras objects (layers, metrics, etc) that are too specialized to be added to core Keras, but that receive the same level of polish and backwards compatibility guarantees as the rest of the Keras API and that are maintained by the Keras team itself (unlike TFAddons).
+KerasCV can be understood as a horizontal extension of the Keras API: they're new first-party Keras 
+objects (layers, metrics, etc) that are too specialized to be added to core Keras, but that receive 
+the same level of polish and backwards compatibility guarantees as the rest of the Keras API and that 
+are maintained by the Keras team itself (unlike TFAddons).
 
-## Under Construction
-KerasCV is a relatively new project.  Expect regular updates and changes in the near future.
+# No Goals
+- **KerasCV is not a repository of blackbox end-to-end solutions (like TF Hub or ModelGarden).**
```

Comment:
II think that this is a little bit too bold claim as models garden rely also on TFAddons (reusable?) components:

https://github.com/tensorflow/models/blob/master/official/requirements.txt#L13

But you are also aware that we historically had many reusable components duplication over time between TFAddons and Model Gardens without reusing TFAddons reusable components (we have many public tickets/threads on this topic).

/cc @seanpmorgan @fsx950223

Reply:
model garden was using some of the TFA optimizer and utils. 

For code sharing and duplication, I think if certain methods and useful but not worth a public API, it is reasonable to fork/copy or keep a slight different version in own code base for simplicity.

Reply:
I agree, I don't see duplicated code as an issue.  I see it as a necessity in providing Keras users with a coherent API for completing CV tasks.

Reply:
I think that it is ok to duplicate or copy private utils but I am not talking about this.
I think that all we agree that Model garden is not exactly a library but my point was not really about private utils but often about components like loss, activation etc.. that is hard to claim as `private`.
I have many example tickets/threads on github accumulated over time but if you want to have just a sample you could check:
https://github.com/tensorflow/models/pull/8962#pullrequestreview-457058720

I am ok that nobody want to depend on anything else external but IMHO by a community point of view there is a difference between an ecosystem and a collection of repos.

Reply:
> model garden was using some of the TFA optimizer and utils.   For code sharing and duplication, I think if certain methods and useful but not worth a public API, it is reasonable to fork/copy or keep a slight different version in own code base for simplicity.

Personally, I also see no issue replicating components that may exist in model garden.  

I.E. cutmix & mixup,  both exist in model garden or TF Hub, but we still want to offer them as a first class preprocessing layer the Keras users

Reply:
Yes I am totally ok with this and also with copy adapting these from model garden.  I have posted many pointers to model gardes in many tickets on this repo so I am totally ok.
The only thing is that I want to avoid to create confusion on where contribute what for new layers. 
But if the goal is to build a competitive community or a brand new ecosystem I am ok.

Reply:
A small additional note. Are we going to collect here higher level API in the style of [TF Object detection API](https://tensorflow-object-detection-api-tutorial.readthedocs.io/en/latest/)?

This kind of API were quite popular on stackoverflow, our forum and on github.